src/visual/GameStateVisualizer.cs

```diff
@@ -1,12 +1,58 @@
 using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
 
 namespace tcg
 {
   static class GameStateVisualizer
   {
-    static string Visualize(GameState state)
+    public static string OutputBoard(Player currPlayer){
+        string toReturn = "";
+        
+        Hero currHero = currPlayer.Hero;
+
+        toReturn += string.Format("Player {0} {1} HP\n", currPlayer.Id, currHero.HP);
+        //toReturn += string.Format("Player {0} {1} HP\n", currPlayer.name, currHero.HP); //Когда будет готов Player.name
+
+        List<Card> activeCards = currPlayer.ActiveCards;
+        for (int j=0; j<activeCards.Count; j++){
+          Card currCard = activeCards[j];
+
+          toReturn += string.Format(currCard.ToString());
+        }
+        toReturn += "---------------------------------------\n";
+
+        return toReturn;
+    }
+
+    public static string OutputHand(Player currPlayer){
+      string toReturn = "";
+
+      List<Card> cardsInHand = currPlayer.CardsInHand;
+
+      for (int i =0; i< cardsInHand.Count; i++){
+        Card currCard = cardsInHand[i];
+
+          toReturn += currCard.ToString();
+      }
+
+      return toReturn;
+    }
+    public static string Visualize(GameState state)
```

Comment:
Let's do everything in the one way
```suggestion
    public static string GameStateToString(GameState state)
```